en-US/Docs/PinMappingsMBM.md

```diff
@@ -167,15 +167,21 @@ using Windows.Devices.I2c;
 
 public async void I2C()
 {
-	var settings = new I2cConnectionSettings(0x40); /* 0x40 is the I2C device address   */
-	settings.BusSpeed = I2cBusSpeed.FastMode;       /* FastMode = 400KHz                */
+    // 0x40 is the I2C device address
+    var settings = new I2cConnectionSettings(0x40);
 
-          */
-	I2cDevice Device = await I2cDevice.GetDefault(settings);    /* Create an I2cDevice with the default controller and the specified I2C settings */
+    // FastMode = 400KHz
+    settings.BusSpeed = I2cBusSpeed.FastMode;
```

Comment:
Let's keep these examples minimal and remove the line that sets BusSpeed to FastMode.